src/languages/en.ts

```diff
@@ -1420,6 +1420,7 @@ const translations = {
         dates: 'Dates',
         rates: 'Rates',
         submitsTo: ({name}: SubmitsToParams) => `Submits to ${name}`,
+        waitingForSubmitterToFixViolations: 'Waiting for you to fix the issue(s). Your admins have restricted submission of expenses that have violations.',
```

Comment:
@jamesdeanexpensify is asking for a small copy update here @abzokhattab. 

'Waiting for you to fix the issue(s). Your admins have restricted submission of expenses with violations.',